src/translations/en.json

```diff
@@ -1417,7 +1417,7 @@
         },
         "scene": {
           "caption": "Scenes",
-          "description": "Manage scenes",
+          "description": "Capture device states and easily activate them later",
```

Comment:
Was just thinking whether instead of "activate" the word "recall" might make it even clearer.